# Patches were sent upstream to maintainer
# as of 23 September 2009, by Jesse Farinacci <jieryn@gmail.com>

--- x3270-3.3/screen.c
+++ x3270-3.3/screen.c
@@ -2322,12 +2322,14 @@
 void
 screen_flip(void)
 {
+#if defined(X3270_DBCS) /*[*/
 	/* Flip mode is broken in the DBCS version. */
 	if (!dbcs) {
 		flipped = !flipped;
 
 		action_internal(PA_Expose_action, IA_REDRAW, CN, CN);
 	}
+#endif /*]*/
 }
 
 /*
